bundles/org.openhab.binding.semsportal/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="semsportal"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="station">
+		<label>SEMSPortal Power Station</label>
+		<description>A Power Station is the GoodWE converter that is connected through the internet with the SEMSPortal.
+			Configuration will only work if you have used this account at least once in the portal itsself.</description>
+
+		<channels>
+			<channel id="online_state" typeId="online_state"/>
+			<channel id="last_update" typeId="last_update"/>
+			<channel id="current_output" typeId="current_output"/>
+			<channel id="today_total" typeId="today_total"/>
+			<channel id="month_total" typeId="month_total"/>
+			<channel id="overall_total" typeId="overall_total"/>
+			<channel id="today_income" typeId="today_income"/>
+			<channel id="total_income" typeId="total_income"/>
+		</channels>
+
+		<config-description>
+
+			<parameter name="username" type="text" required="true">
+				<label>Username</label>
+				<description>Account name of the SEMS portal</description>
+			</parameter>
+
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
```

Comment:
```suggestion
			<parameter name="password" type="text" required="true">
			<context>password</context>
				<label>Password</label>
```

Will hide the password when this is added.

Reply:
Great, was looking for an attribute to do that. modified.